.github/workflows/integration.yml

```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   integration-py3:
     container:
-      image: nvcr.io/nvidia/pytorch:21.12-py3  # CUDA 11.5
+      image: nvcr.io/nvidia/pytorch:21.10-py3  # CUDA 11.4
```

Comment:
this image wouldn't work with line 37 `torch==1.11.0+cu115` could you test?

Reply:
updated in the latest commit.

Thanks.

Reply:
this ci config will be used when this PR is merged to dev, so https://github.com/Project-MONAI/MONAI/runs/5570109568?check_suite_focus=true is still out-of-date, I think you can test locally of the commands here or have a temporary test like this one https://github.com/Project-MONAI/MONAI/blame/3b771bb8774b89e5ea630b9782070415a979202c/.github/workflows/integration.yml#L6-L8

Reply:
OK, I already tested below commands in the 21.10 docker locally:
```
root@apt-sh-ai:/workspace/data/medical/MONAI# pip install torch==1.11.0+cu113 torchvision==0.12.0+cu113 -f https://download.pytorch.org/whl/torch_stable.html
Looking in indexes: https://pypi.org/simple, https://pypi.ngc.nvidia.com
Looking in links: https://download.pytorch.org/whl/torch_stable.html
Requirement already satisfied: torch==1.11.0+cu113 in /opt/conda/lib/python3.8/site-packages (1.11.0+cu113)
Requirement already satisfied: torchvision==0.12.0+cu113 in /opt/conda/lib/python3.8/site-packages (0.12.0+cu113)
Requirement already satisfied: typing-extensions in /opt/conda/lib/python3.8/site-packages (from torch==1.11.0+cu113) (3.10.0.2)
Requirement already satisfied: requests in /opt/conda/lib/python3.8/site-packages (from torchvision==0.12.0+cu113) (2.26.0)
Requirement already satisfied: numpy in /opt/conda/lib/python3.8/site-packages (from torchvision==0.12.0+cu113) (1.21.2)
Requirement already satisfied: pillow!=8.3.*,>=5.3.0 in /opt/conda/lib/python3.8/site-packages (from torchvision==0.12.0+cu113) (9.0.1)
Requirement already satisfied: charset-normalizer~=2.0.0 in /opt/conda/lib/python3.8/site-packages (from requests->torchvision==0.12.0+cu113) (2.0.0)
Requirement already satisfied: idna<4,>=2.5 in /opt/conda/lib/python3.8/site-packages (from requests->torchvision==0.12.0+cu113) (3.1)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in /opt/conda/lib/python3.8/site-packages (from requests->torchvision==0.12.0+cu113) (1.26.7)
Requirement already satisfied: certifi>=2017.4.17 in /opt/conda/lib/python3.8/site-packages (from requests->torchvision==0.12.0+cu113) (2021.5.30)
root@apt-sh-ai:/workspace/data/medical/MONAI# export CUDA_VISIBLE_DEVICES=$(python -m tests.utils); python -m tests.test_integration_classification_2d
----------
Epoch 1/4
epoch 1 average loss:0.7807
saved new best metric model
current epoch 1 current AUC: 0.9978 current accuracy: 0.9538 best AUC: 0.9978 at epoch 1
----------
Epoch 2/4
epoch 2 average loss:0.1623
...
```
And the test GPU memory is as expected:
<img width="561" alt="image" src="https://user-images.githubusercontent.com/27533405/158636295-45eca6f5-2ff9-473f-9b57-297508df6a88.png">

Thanks.